R/pandoc.R

```diff
@@ -624,6 +605,42 @@ is_highlightjs <- function(highlight) {
   !is.null(highlight) && (highlight %in% c("default", "textmate"))
 }
 
+resolve_highlight <- function(highlight, supported = highlighters()) {
+  is_supported <- TRUE
+  # for backward compatibility, partial match still need to work
+  highlight <- tryCatch(
+    error = function(e) { is_supported <<- FALSE; highlight },
+    match.arg(highlight, supported)
+  )
+  if (is_supported) return(highlight)
```

Comment:
I did this change to bring back the partial matching feature that was there when `match.arg()` was used. Meaning
this would work. 
````yaml
output:
  html_document: 
    highlight: breez
````
It was not following my other PR.